### DISCUSSION TITLE
Change owner of a hobby team account

### DISCUSSION THREAD

**Discussion by shawnmclean** [vercel/community#4511] [Account Issue] [answered]
### Question

How can I change the owner of a hobby team?

Do I have to pay to add the person and then remove myself? (https://github.com/orgs/vercel/discussions/2956#discussioncomment-6254049)

This isn't feasible, its going to cost us $40 just to transfer ownership of a team?

What happens if I'm the only owner and remove myself from the team, can I add another person?

**Comment by amyegan** [comment, marked as answer]
To transfer team ownership, you would need to have both people included on a paid Pro plan. But it is possible to transfer project and domain ownership.

You can go to the team's settings page and transfer your projects to another team or account. You can just [transfer domains](https://vercel.com/docs/projects/overview#transferring-domains) and delete the old project if the other person wants to redeploy the code to a new project on their own account. 

If the projects stay within the [Fair Use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#) for a Hobby plan, then the projects can be moved to a personal account instead of a team. Remember that Hobby accounts are restricted to non-commercial personal use only, so a Pro plan is needed if the projects are used for financial gain. More info at https://vercel.com/docs/limits/fair-use-guidelines#commercial-usage
